components/Image/FullWidthImage.tsx

```diff
@@ -0,0 +1,53 @@
+import { useMemo } from 'react';
+import { cnb } from 'cnbuilder';
+import { getSbImageSize } from '@/utilities/getSbImageSize';
+import { getImageSources } from '@/utilities/getImageSources';
+import { type SbImageType } from '@/components/Storyblok/Storyblok.types';
+import * as styles from './Image.styles';
+
+export type FullWidthImageProps = SbImageType & React.HTMLAttributes<HTMLImageElement> & {
+  classPrefix?: string;
+  visibleVertical?: styles.VisibleVerticalType;
+  visibleHorizontal?: styles.VisibleHorizontalType;
+};
+
+export const FullWidthImage = ({
+  filename,
+  alt,
+  classPrefix,
+  visibleHorizontal,
+  visibleVertical,
+  className,
+}: FullWidthImageProps) => {
+  const { width: originalWidth, height: originalHeight } = getSbImageSize(filename);
+
+ // Get corresponding image sources for responsive images
+  const imageSources = useMemo(() => {
+    return getImageSources(filename, originalWidth);
+  }, [originalWidth, filename]);
+
+  return (
+    <div className={cnb('su-media', classPrefix && `${classPrefix}__media`, className)}>
```

Comment:
Leaving all the prefixes and custom classes in for now so we don't miss adding any styles later